include/swift/Basic/LLVM.h

```diff
@@ -42,8 +42,9 @@ namespace llvm {
   template<typename T> class MutableArrayRef;
   template<typename T> class TinyPtrVector;
   template<typename T> class Optional;
-  template <typename PT1, typename PT2> class PointerUnion;
-  template <typename PT1, typename PT2, typename PT3> class PointerUnion3;
+  template <typename ...PTs> class PointerUnion;
+  template <typename PT1, typename PT2, typename PT3>
+  using PointerUnion3 = PointerUnion<PT1, PT2, PT3>;
```

Comment:
Is anything still depending on this? (The purpose of this file is just to pull some LLVM things into the Swift namespace for convenience.)

Reply:
Yeah `ASTNode` uses that, but I think this is as good a time as any to just use `PointerUnion` instead of `PointerUnion3`.